src/script/lua_api/l_object.cpp

```diff
@@ -722,15 +722,15 @@ int ObjectRef::l_set_detach(lua_State *L)
 
 	int parent_id = 0;
 	std::string bone = "";
-	v3f position;
-	v3f rotation;
+	v3f position = v3f(0, 0, 0);
```

Comment:
interesting, I can find at least 12 instances of it being initialised to default in the file